openmmml/models/anipotential.py

```diff
@@ -81,8 +91,27 @@ def addForces(self,
         includedAtoms = list(topology.atoms())
         if atoms is not None:
             includedAtoms = [includedAtoms[i] for i in atoms]
-        elements = [atom.element.symbol for atom in includedAtoms]
-        species = model.species_to_tensor(elements).unsqueeze(0)
+        # elements = [atom.element.symbol for atom in includedAtoms]
+        #species = model.species_to_tensor(elements).unsqueeze(0)
+        species = torch.tensor([[atom.element.atomic_number for atom in includedAtoms]])
+
+        if implementation == 'nnpops':
+            try:
+                from NNPOps import OptimizedTorchANI
+                #device = torch.device('cuda') # nnpops doesn't need cuda necessarily
+            except Exception as e:
+                print(f"failed to import `nnpops` with error: {e}")
+            # species = torch.tensor([[atom.element.atomic_number for atom in includedAtoms]], device=device) # with nnpops, one needs `atomic_number`
+            try:
```

Comment:
We shouldn't hardcode any references to CUDA.  If your simulation uses the CUDA platform, the OpenMM-Torch plugin will automatically move the model to the correct device.  And if you use a different platform, this will break it.

Reply:
I am not totally convinced this is the case; if i run a short simulation without explicitly placing `model.to(device)`, then i see 0.03s/step, but if i leave the code alone, i see ~0.007s/step.

Reply:
perhaps this is a bug?

Reply:
ah, scratch this. my test was faulty. it appears to work now